src/main/resources/application.yaml

```diff
@@ -31,3 +31,5 @@ spring:
 #          lob:
 #            # silence the 'wall-of-text' - unnecessary exception throw about blob types
 #            non_contextual_creation: true
+  http:
+    log-request-details: true
```

Comment:
what does this do?

Reply:
hmm, checked with and without and passing around some dummy `Authorization` header - didn't notice a difference. but perhaps it's best to remove what do you think?

the description about the flag is logged:
```bash
DEBUG [http-nio-4550-exec-1] org.springframework.web.servlet.DispatcherServlet:538: enableLoggingRequestDetails='true': request parameters and headers will be shown which may lead to unsafe logging of potentially sensitive data
```